xbmc/pvr/dialogs/GUIDialogPVRRecordingInfo.cpp

```diff
@@ -59,8 +59,11 @@ bool CGUIDialogPVRRecordingInfo::OnClickButtonPlay(const CGUIMessage& message)
     Close();
 
     if (m_recordItem)
-      CServiceBroker::GetPVRManager().Get<PVR::GUI::Playback>().PlayRecording(
-          *m_recordItem, true /* check resume */);
+    {
+      CGUIPVRRecordingsPlayActionProcessor proc{*m_recordItem};
+      if (proc.Process() && proc.UserCancelled())
```

Comment:
Process() is always mandatory right? I think combining both conditions make the code simpler but hides the intention.
Open() should be only called if canceled by user isnt it?
Why not simply call process on a line and only check for cancel in the if condition?

Reply:
Fair. I will change that.